### ISSUE TITLE
Insert and Delete missing from query builder in typescript

### ISSUE THREAD

**Issue by Julius-Bendt** [sutandojs/sutando#56]
Hi there! 
First of all - thanks for this package! I'm happy that something like eloquent exists for Node 

Given the examples in https://sutando.org/guide/query-builder.html#insert-statements:

```ts
const db = sutando.connection();

await db.table('users').insert({
    email: 'kayla@example.com',
    votes: 0
});

const deleted = await db.table('users').where('votes', '>', 100).delete();
```

I get the following error: `Property 'insert/delete' does not exist on type 'AnyQueryBuilder'`.

I've tried to typecast as follows: `const db = sutando.connection() as QueryBuilder<any, any>;`  but with the same result.

**Comment by kiddyuchina** [comment]
Thanks for the feedback.  it has been resolved in v1.6.2.

**Comment by Julius-Bendt** [comment]
Thanks for the quick answer and fix! :D
